src/fragments/lib/storage/js/upload.mdx

```diff
@@ -61,49 +63,80 @@ const result = await Storage.put('test.txt', 'File content', {
 Other options available are:
 
 ```javascript
-Storage.put('test.txt', 'My Content', {
-    acl: 'public-read', // https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#upload-property
-    cacheControl: 'no-cache', // (String) Specifies caching behavior along the request/reply chain
-    contentDisposition: 'attachment', // (String) Specifies presentational information for the object
-    expires: new Date().now() + 60 * 60 * 24 * 7, // (Date) The date and time at which the object is no longer cacheable. ISO-8601 string, or a UNIX timestamp in seconds
-    metadata: { key: 'value' }, // (map<String>) A map of metadata to store with the object in S3.
-})
+Storage.put("test.txt", "My Content", {
+  acl: "public-read", // https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#upload-property
+  cacheControl: "no-cache", // (String) Specifies caching behavior along the request/reply chain
+  contentDisposition: "attachment", // (String) Specifies presentational information for the object
+  expires: new Date().now() + 60 * 60 * 24 * 7, // (Date) The date and time at which the object is no longer cacheable. ISO-8601 string, or a UNIX timestamp in seconds
+  metadata: {key: "value"}, // (map<String>) A map of metadata to store with the object in S3.
+  resumable: true, // (Boolean) Allows uploads to be paused and resumed
+});
+```
+
+## Pause and resume uploads
+
+Passing the `resumable: true` option to the Put API returns an object with the following methods: `pause` , `resume` , `abort`.
+
+```javascript
+const upload = await Storage.put(file.name, file, {
+  resumable: true,
+});
+
+upload.pause();
+
+upload.resume();
+
+upload.abort();
+```
+
+When a page refresh occurs during the upload, re-initializing the upload with the same file will continue from previous progress point.
+
+```javascript
+const upload = await Storage.put(file.name, file, {
+  resumable: true,
+});
+
+// This duplicate uploads will resume the original upload.
+const duplicateUpload = await Storage.put(file.name, file, {
+  resumable: true,
+});
```

Comment:
```suggestion
```javascript
const upload = await Storage.put(file.name, file, {
  resumable: true,
});

// Browser refreshes ...

// This duplicate uploads will resume the original upload.
const duplicateUpload = await Storage.put(file.name, file, {
  resumable: true,
});
```